modules/postgresql/README.md

```diff
@@ -6,69 +6,72 @@
+| zone | The zone for the master instance, it should be something like: `a`, `c`. | string | n/a | yes |
 
 ## Outputs
 
 | Name | Description |
 |------|-------------|
-| generated_user_password | The auto generated default user password if not input password was provided |
-| instance_address | The IPv4 addesses assigned for the master instance |
-| instance_connection_name | The connection name of the master instance to be used in connection strings |
-| instance_first_ip_address | The first IPv4 address of the addresses assigned. |
-| instance_name | The instance name for the master instance |
-| instance_self_link | The URI of the master instance |
-| instance_server_ca_cert | The CA certificate information used to connect to the SQL instance via SSL |
-| instance_service_account_email_address | The service account email address assigned to the master instance |
-| read_replica_instance_names | The instance names for the read replica instances |
-| replicas_instance_connection_names | The connection names of the replica instances to be used in connection strings |
-| replicas_instance_ip_addresses | The IPv4 addresses assigned for the replica instances |
-| replicas_instance_self_links | The URIs of the replica instances |
-| replicas_instance_server_ca_certs | The CA certificates information used to connect to the replica instances via SSL |
-| replicas_instance_service_account_email_addresses | The service account email addresses assigned to the replica instances |
+| generated\_user\_password | The auto generated default user password if not input password was provided |
+| instance\_address | The IPv4 addesses assigned for the master instance |
+| instance\_connection\_name | The connection name of the master instance to be used in connection strings |
+| instance\_first\_ip\_address | The first IPv4 address of the addresses assigned. |
+| instance\_name | The instance name for the master instance |
+| instance\_self\_link | The URI of the master instance |
+| instance\_server\_ca\_cert | The CA certificate information used to connect to the SQL instance via SSL |
+| instance\_service\_account\_email\_address | The service account email address assigned to the master instance |
+| read\_replica\_instance\_names | The instance names for the read replica instances |
+| replicas\_instance\_connection\_names | The connection names of the replica instances to be used in connection strings |
+| replicas\_instance\_ip\_addresses | The IPv4 addresses assigned for the replica instances |
+| replicas\_instance\_self\_links | The URIs of the replica instances |
+| replicas\_instance\_server\_ca\_certs | The CA certificates information used to connect to the replica instances via SSL |
+| replicas\_instance\_service\_account\_email\_addresses | The service account email addresses assigned to the replica instances |
 
```

Comment:
I'm aware of this changes because I reviewed #26 however you should drop the changes generated by the lint at this time.
We should discuss the subject of the lint on #26